app/controllers/restaurants_controller.rb

```diff
@@ -0,0 +1,30 @@
+class RestaurantsController < ApplicationController
+
+  include Response
+  include ActiveModel
+  def index
+    @restaurants = Restaurant.all
+    # Cuisine.find(@restaurants.cuisine_id)
+    # render :json => @restaurants.to_json(:except => [:created_at, :updated_at], include: 'cuisine' )
+    render :json => @restaurants.to_json(:except => [:created_at, :updated_at, :cuisine_id], :include => {:reviews => {:except => [:created_at, :updated_at, :restaurant_id]}, :cuisine => {:except => [:created_at, :updated_at]}})
+  end
+
+  def averageRating
+    @avg = nil
```

Comment:
we prefer to use full-length name - average instead of avg, element instead of el

Reply:
Also, prefer local variables to instance variables (without the @)
Instance variables are useful when you want to keep a cache of some heavy operation that's used several times for the same instance, but otherwise we can just use local variables